### PR TITLE
CUMULUS-1 Merge master at aebdfc582 to feature-rds

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-1: ](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

## Changes

* Merging master @ aebdfc582 to feature-rds

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

***DO NOT SQUASH***

### CODE REVIEW

File: docs/tasks.md [hunk 1]
```diff
@@ -149,3 +149,12 @@ Updates CMR metadata to set access constraints
 
 - Schemas: See this module's [schema definitions](https://github.com/nasa/cumulus/tree/master/tasks/update-cmr-access-constraints#readme/schemas).
 - Resources: [npm](https://npmjs.com/package/@cumulus/update-cmr-access-constraints) | [source](https://github.com/nasa/cumulus) | [web](https://github.com/nasa/cumulus/tree/master/tasks/update-cmr-access-constraints#readme)
+
+---
+
+### [@cumulus/update-granules-cmr-metadata-file-links](https://github.com/nasa/cumulus/tree/master/tasks/update-granules-cmr-metadata-file-links)
+
+Update cmr metadata files with correct online access urls and etags and transfer etag info to granules' CMR files
```
Comment (botanical) [inline docs/tasks.md:157]: Can we capitalize `CMR`?

File: example/cumulus-tf/outputs.tf [hunk 2]
```diff
@@ -35,3 +35,11 @@ output "report_granules_sns_topic_arn" {
...
   value = module.cumulus.report_pdrs_sns_topic_arn
 }
+
+output "lzards_backup_task" {
+  value = module.cumulus.lzards_backup_task
+}
+
+output "move_granules_task" {
+  value = module.cumulus.move_granules_task
+}
```
Comment (botanical) [inline example/cumulus-tf/outputs.tf:45]: Newline

**Review by botanical — APPROVED** [review]
Minor nits but lgtm

File: docs/tasks.md (hunk 1 above)
Comment (Jkovarik) [reply docs/tasks.md:157]: We can, however this is a direct merge of already approved PRs from master->feature branch.    I can put a PR in against master re: this later.

**Pull request merged**